tests/integration/test_tunnels_background.py

```
@@ -0,0 +1,113 @@
+
+
+def test_tunnels_background_detaches(shed_server, test_shed_name):
+    server = shed_server
+    name = test_shed_name
+    server.create(name)
+
+    local_port = _free_local_port()
+    pid = None
+    try:
+        t0 = time.monotonic()
+        r = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)